assets/js/components/surveys/CurrentSurvey.js

```diff
@@ -151,23 +162,29 @@ const CurrentSurvey = () => {
 		}
 	}, [ hasSentCompletionEvent, sendSurveyEvent, triggeredCompletion ] );
 
+	useMount( () => {
+		setAnimateSurvey( true );
+	} );
```

Comment:
@asvinb why not use `true` as the default value for the `const [ animateSurvey, setAnimateSurvey ] = useState( false );`?

Reply:
@eugene-manuilov When testing I noticed that the animation does not fire up when in the WordPress dashboard. The popup just shows up without any animation. So, we make sure it's only when the component is mounted, then it fires the animation.